app/Components/Common/LabelAccordian/index.js

```diff
@@ -1,37 +1,84 @@
-import React, { useState } from 'react';
+import React, { useState, useContext, useEffect } from 'react';
 import { View, Text, StyleSheet, Image, TouchableOpacity, ScrollView, FlatList } from 'react-native';
 import { Colors, Typography } from '_styles';
 import { arrow_down, arrow_up } from '../../../assets/';
 import { Input, Label } from '../';
+import { Accordian } from '../'
+import { store } from '../../../Actions/store';
+import { addSpeciesAction, updateLastScreen, getInventory, updatePlantingDate } from '../../../Actions'
 
+const LabelAccordian = ({ data, onPressRightText, isEditShow, plantingDate, status }) => {
+    const { state } = useContext(store);
 
-const LabelAccordian = ({ data, onPressRightText }) => {
+    const [species, setSpecies] = useState([])
 
-    const renderSubSpecie = (item) => (
-        <TouchableOpacity style={styles.oneSpecieCont}>
-            <Text style={styles.label}>{item.nameOfTree ? item.nameOfTree : 'Species'}</Text>
-            <View style={styles.treeCountCont}>
-                <>
-                    <Text style={styles.treeCount}>{item.treeCount}</Text>
-                    <Text style={styles.trees}>Trees</Text>
-                </>
-            </View>
-        </TouchableOpacity>
+    useEffect(() => {
+        if (data) setSpecies(data)
+    }, [])
```

Comment:
This makes it only run once in start why not just do and remove useEffect completely.

`const [species, setSpecies] = useState(data ? data : [])`

Reply:
fixed